src/sentry/models/savedsearch.py

```diff
@@ -27,7 +27,7 @@ def as_choices(cls):
         return (
             (cls.DATE, _("Last Seen")),
             (cls.NEW, _("First Seen")),
-            (cls.PRIORITY, _("Priority")),
```

Comment:
Hm I changed this with the intention of starting to write trends in saved searches instead of priority, but we can do that after the FE is changed.

Reply:
It might be best to have both priority and trends as options here. The frontend decides which options to show ([see here](https://github.com/getsentry/sentry/blob/5fe143b07a16cea81dedc708692b9c48ad0ee26c/static/app/components/modals/savedSearchModal/savedSearchModalContent.tsx#L24-L30)), so I don't think having the extra option here would affect the UI in any way. That way you can modify the frontend to send trends and it won't break (I believe this gets used in the validators when creating a saved search?)